channels/serializers.py

```diff
@@ -269,7 +238,7 @@ def create(self, validated_data):
 
         api.add_post_subscription(post.id)
 
-        changed = _apply_vote(post, validated_data, False)
+        changed = apply_post_vote(post, validated_data)
```

Comment:
I think there's a possible race condition here under multiple workers where the index update for the upvote here gets processed _before_ the full post task index is processed on the `create_post` call above. I would expect since you're doing a partial ES update that call has potential to fail where the document to update doesn't exist. If it's not practical to address that right now, we may want to catch those errors so we don't get a bunch of noise in production.

I have some thoughts on how to handle that race condition, but I'm not sure if that's something we want to chew off right now.

Reply:
Yup, that would be possible. I can create an issue to look into this